dao/OrderRepository.go

```diff
@@ -78,11 +78,28 @@ func (r *OrderPostgres) GetActiveOrderFromDB(id int) (Order, error) {
 	}
 	return Ord, nil
 }
+func (r *OrderPostgres) GetOrderFromDB(id int) (Order, error) {
+	var Ord Order
+
+	insertValue := `Select delivery_service_id,id,courier_id,delivery_time,customer_address,status,order_date,restaurant_address,picked from delivery where id = $1`
+	get, err := r.db.Query(insertValue, id)
+	if err != nil {
+		log.Println("Error with getting order by id: " + err.Error())
+		return Order{}, err
+	}
+
+	for get.Next() {
+		var order Order
+		err = get.Scan(&order.IdDeliveryService, &order.Id, &order.IdCourier, &order.DeliveryTime, &order.CustomerAddress, &order.Status, &order.OrderDate, &order.RestaurantAddress, &order.Picked)
```

Comment:
Why do you use r.db.Query? If you expect to get one object, you can use r.db.QueryRow and you don't need to use then get.Next().